zwave_js/DOCS.md

```diff
@@ -138,6 +138,17 @@ This option sets the log level of Z-Wave JS. Valid options are:
 If no `log_level` is specified, the log level will be set to the level set in
 the Supervisor.
 
+### Option `log_to_file`
+
+When this option is enabled, logs will be written to the `/addon_configs/core_zwave_js`
```

Comment:
How can the user access this folder?

Reply:
shouldn't it just be available? https://developers.home-assistant.io/blog/2023/11/06/public-addon-config/

Reply:
shouldn't it just be available? https://developers.home-assistant.io/blog/2023/11/06/public-addon-config/

Reply:
Well, the user will typically need another add-on to access files, like the file editor, samba, or ssh add-ons. There's no user interface in this add-on to access those files. Do we need to explain that?